garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -48,6 +54,11 @@ local function IdentifyBody(ply, rag)
    local finder = ply:Nick()
    local nick = CORPSE.GetPlayerNick(rag, "")
    local traitor = (rag.was_role == ROLE_TRAITOR)
+   
+   -- opportunity for addons to introduce ways for traitors(ideally) to interrupt corpse identification.
+   if hook.Run("TTTInterruptCorpseIdentify", ply, rag, traitor) then
+      return
+   end

```

Comment:
the `traitor` variable is one of the locals, which is used in the hook call, so it only the other locals should be moved below the hook call.